### DISCUSSION TITLE
Mediation organization

### DISCUSSION THREAD

**Discussion by EEanalyst** [OP-TED/eForms-SDK#288] [Q&A] [answered]
Why is OPT-301-Part-Mediator marked mandatory for notices 4 and 5 in Fields.json? And at the same time OPT-301-Lot-Mediator isn't mandatory.

As per SDK "Table 10. Role/subrole presence per notice" in here: https://docs.ted.europa.eu/eforms/1.3/schema/parties.html, It says that mediation organization is existing mandatory. Meaning it should be used only when such organization exists and in EE there isn't one. We only have review organization.

If we leave out cac:MediationParty in cac:AppealTerms we get an error saying "ID – Mediator (OPT-301-Part-Mediator) is mandatory for a notice with subtype 4".  How can we get around this?

We are developing on version 1.3.2.

**Comment by YvesJo** [comment, marked as answer]
Indeed Mediator is Mandatory only if it exists. Rule will be fixed with a next release.
